zerver/lib/bugdown/__init__.py

```diff
@@ -754,11 +754,14 @@ def handleMatch(self, match):
 def make_emoji(codepoint, display_string):
     # type: (Text, Text) -> Element
     src = '/static/generated/emoji/images/emoji/unicode/%s.png' % (codepoint,)
+    # Replace underscore in emoji's title with space
+    title = display_string[1:-1].replace("_", " ")
+
     elt = markdown.util.etree.Element('img')
     elt.set('src', src)
     elt.set('class', 'emoji')
     elt.set("alt", display_string)
```

Comment:
ah sorry, i had misunderstood what we were dealing with!

looked through those links and it seems like `aria-label="name_of_emoji"` would be enough of an addition, as we are dealing with an image already.

the second link seems to describe a technique that covers both screenreaders & people with processing/recognition issues: do we want to have the "hover over emoji to get description" as an optional feature?